include/GafferUI/PlugAdder.h

```diff
@@ -37,24 +37,32 @@
 #ifndef GAFFERUI_PLUGADDER_H
 #define GAFFERUI_PLUGADDER_H
 
+#include "GafferUI/ConnectionCreator.h"
 #include "GafferUI/StandardNodeGadget.h"
 
 namespace GafferUI
 {
 
-class PlugAdder : public Gadget
+class PlugAdder : public ConnectionCreator
 {
 
 	public :
 
-		IE_CORE_DECLARERUNTIMETYPEDEXTENSION( GafferUI::PlugAdder, PlugAdderTypeId, Gadget );
+		IE_CORE_DECLARERUNTIMETYPEDEXTENSION( GafferUI::PlugAdder, PlugAdderTypeId, ConnectionCreator );
 
 		PlugAdder( StandardNodeGadget::Edge edge );
 		virtual ~PlugAdder();
 
 		virtual Imath::Box3f bound() const;
 
-		void updateDragEndPoint( const Imath::V3f position, const Imath::V3f &tangent );
+		/// May be called by the recipient of a drag to set a more appropriate position
```

Comment:
There's no need to repeat the documentation from the base class - it just makes it extremely likely that the two will get out of sync. I think there are a few spots where this applies.